src/main/java/core/basesyntax/FigureColor.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class FigureColor implements Figure, AreaCalculator {
```

Comment:
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```
And add constructor for color here.